src/exception/user-not-found.error.ts

```diff
@@ -0,0 +1,6 @@
+export class UserNotFoundError extends Error {
```

Comment:
Can we make the Expection nomenclature and the class from which it is inherited as in the nestjs document? 

https://docs.nestjs.com/exception-filters#custom-exceptions